migrations/20210611024943_archived-status-notifications-icon-rejection-reasons.sql

```diff
@@ -0,0 +1,18 @@
+INSERT INTO statuses (status) VALUES ('archived');
+
+ALTER TABLE notifications
+    ADD COLUMN icon varchar(20000);
+
+ALTER TABLE mods
```

Comment:
This is fine for now, but we should really move to a ticketing system for rejections, as we usually want to get a response instead of blindly refusing it.

Reply:
Agreed. It's a temporary solution while we make a robust solution that will be permanent